README.md

```diff
@@ -22,6 +22,10 @@ Seq2Pat is developed as a joint collaboration between Fidelity Investments
 and the Tepper School of Business at CMU. Documentation is available at [fidelity.github.io/seq2pat](https://fidelity.github.io/seq2pat).
 
 ## Quick Start
+
+We present two quick start examples as follows. Notice that the input sequences in each example have items 
+represented as strings, while Seq2Pat also allows input to be sequence of integers and the integers must be positive.
+
```

Comment:
possible rephrase;

We present examples for constraint-based sequential pattern mining and dichotomic pattern mining. Sequences can be represented as strings or positive integers.